### DISCUSSION TITLE
Alias for docker_context (due to using Colima)

### DISCUSSION THREAD

**Discussion by mcouthon** [starship/starship#5205] [Q&A]
Hi, I'm using [colima](https://github.com/abiosoft/colima) as my Docker engine, and to make it work well I had to set `$DOCKER_HOST` to `unix://$HOME/.colima/docker.sock`, which causes the `docker_context` inside Starship to present like this

<img width="1327" alt="CleanShot 2023-05-26 at 14 01 13@2x" src="https://github.com/starship/starship/assets/844807/8a6dfcd8-cd70-4546-914f-93e8645e5ad9">

which is obviously not very pretty. I can of course turn it off entirely, but I'd prefer to catch the fact that I'm using colima, and alias the context (like I do for eg my K8s context).

Anyone knows if this is at all possible?

**Comment by mickimnet** [comment]
You could remove the `$context` from the `format` value or exchange it to something hard-coded.

**Reply by mcouthon** [reply]
Yeah, that's what I did. I was just hoping for something more dynamic. Like, if I'm using `colima` to write `colima`, but if I'm using Docker Desktop, write that. Like I did in k8s context aliases:

```toml
[kubernetes.context_aliases]
'arn:aws:eks:(?P<var_region>[\w-]+):(?P<var_account>[\d]+):cluster/(?P<var_cluster>[\w-]+).*' = '$var_cluster'
```

It's really not a big deal though, and if it's not possible I'll live :)
